Week2/homework/maartjes_work.js

```diff
@@ -1,24 +1,22 @@
-'use strict';
-
-const monday = [
-  {
-    name: 'Write a summary HTML/CSS',
+const tasks = monday.concat(tuesday);
+// Add your code here
```

Comment:
`monday` is not defined, the 
`const monday = [` is missing.

Reply:
Once that happens your code runs.  It seems to me that this code was not tested before the PR was created.  You should always, always test your code before sending, and test it again after you have made changes.